src/libcode/vx_tc_util/track_info.cc

```diff
@@ -1016,6 +1026,13 @@ TrackInfo consensus(const TrackInfoArray &tracks,
               << tracks[i].technique_number() << ").\n\n";
       }
 
+      // Get unique diag names
+      // Store diag_names in tavg object
+      for(j=0; j<tracks[i].diag_name().n(); j++) {
+         mlog << Debug(3) << "TrackInfo::consensus(). Found unique diag_name[" << j << "] = " << tracks[i].diag_name()[j] << "\n";
```

Comment:
```suggestion
```
I think I'd just remove this misleading log message. The diag_name will be logged for all input tracks and stating them as being unique is misleading.